src/scales/scale.time.js

```diff
@@ -232,8 +232,9 @@ module.exports = function(Chart) {
 			var timeOpts = me.options.time;
 
 			// We store the data range as unix millisecond timestamps so dataMin and dataMax will always be integers.
-			var dataMin = Number.MAX_SAFE_INTEGER;
-			var dataMax = Number.MIN_SAFE_INTEGER;
+			// Integer constants are from the ES6 spec.
+			var dataMin = Number.MAX_SAFE_INTEGER || 9007199254740991;
```

Comment:
Is there any advantage to keep the constants here compared to simply hard code these values?